datastacks/datastacks_cli.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import shutil
+import yaml
+
+import click
+from click_loglevel import LogLevel
+
+from datastacks.logger import setup_logger
+from datastacks.utils import render_template_components
+
+
+@click.group()
+@click.option("--log-level", "-l", type=LogLevel(), default=logging.INFO)
+def cli(log_level):
+    setup_logger("datastacks", log_level)
+
+@cli.group()
+def generate():
+    pass
+
+@generate.command()
+@click.option("--config-path", "-c", type=str, help="Absolute path to config file on local machine")
```

Comment:
```suggestion
@click.option("--config", "-c", type=str, help="Absolute path to config file on local machine")
```

How about a shorter parameter? We have the help to provide more details. Feel free to ignore & resolve if you don't agree.

Reply:
I generally like being explicit with what's expected but happy to change